Resources/Prototypes/Maps/atlas.yml

```diff
@@ -2,8 +2,8 @@
   id: Atlas
   mapName: Atlas
   mapPath: /Maps/atlas.yml
-  minPlayers: 5
-  maxPlayers: 20
+  minPlayers: 0
```

Comment:
atlas might be pretty dire at extremely low pops but idk. i rarely play at sub 20 playercount so i defer to other folks judgement there

Reply:
atlas should not show up this early, i agree. reach runs itself without engineers at this pop, atlas requires an engineer or for people to break in